src/actions/account.js

```diff
@@ -124,7 +124,7 @@ const checkContractId = () => async (dispatch, getState) => {
         }
 
         try {
-            await (await wallet.getAccount(contract_id)).state();
+            await (wallet.getAccountBasic(contract_id)).state();
```

Comment:
Since `wallet.getAccountBasic()` isn't async and you removed one of the `await` keywords, :hocho: extra parens